.gitignore

```diff
@@ -4,4 +4,5 @@ htmlcov/
 kuulemma/static/bower_components/
 kuulemma/static/dist/
 node_modules/
-env/
\ No newline at end of file
+env/
+.idea
```

Comment:
This could perhaps be removed. It's not something other developers would need. 
But the addition of the empty line at the end of the file is good!